app/templates/community/index.hbs

```diff
@@ -1,122 +1,127 @@
+            for questions with that tag</a>. Please check to see if your question has already been answered before asking a new
+          one.
+        </p>
+        <p>
+          Please note that these resources are not managed or moderated by the Ember Core Team. They are public forums.
+        </p>
+      </div>
+    </div>
+  </section>
+  <section class="margin-top-medium" aria-labelledby="section-latest-news">
+    <h2>Stay Up to Date with the Latest News</h2>
     <p>
-      The Ember.js source is hosted on <a href="https://github.com/emberjs/ember.js/">Github</a>. To contribute patches, create a fork of the project on GitHub and submit a pull request. Please be sure to include unit tests and documentation for any new features you add. See the <a href="https://github.com/emberjs/ember.js/blob/master/CONTRIBUTING.md">full guidelines</a> for more information.
+      <a href="https://the-emberjs-times.ongoodbits.com/">Ember.js Times</a> &mdash; follow the progress of new features in
+      the Ember ecosystem, requests for community input (RFCs), and calls for contributors
     </p>
-  </div>
-</section>
-
-<section class="section-split">
-  <div class="section-split-item text-center">
-    <h2 class="text-center">Feeling Lonely?<br>Join a Meetup</h2>
-
-    <img alt="Meetup" src="/images/community/meetup.png">
     <p>
-      Joining an Ember.js meetup is a great way to learn more about Ember, meet other developers, find jobs, etc. Any and all things Ember and local.
+      <a href="http://emberweekly.com/">Ember Weekly</a> &mdash; a curated list of Ember learning resources (podcasts,
+      videos, blog posts, books, and more)
     </p>
-
-    {{#link-to "community.meetups" class="es-button"}}
-      Find a meetup
-    {{/link-to}}
-  </div>
-
-  <div class="section-split-item text-center">
-
-    <h2 class="text-center">Met The Mascots?<br>You'll Love Them!</h2>
-
-    <img alt="Campster" src="/images/community/campster.png">
     <p>
-      Tomster and Zoey have been crafted with love, and are constantly getting makeovers. Events, User Groups, Countries&mdash;sky's the limit!
+      <a href="https://emberjs.com/blog/">Official Ember Blog</a> &mdash; big announcements like new Ember.js version
+      release notes or State of the Union information
     </p>
-    {{#link-to "mascots" class="es-button"}}
-      Meet the Mascots
-    {{/link-to}}
-  </div>
-</section>
-
-<section class="meetup-video">
-  <div class="video">
-    <div class="embedded-video">
-      <iframe
-        width="640"
-        height="360"
-        src="https://www.youtube.com/embed/rY5D38RQoEg?rel=0"
-        frameborder="0"
-        allowfullscreen
-      >
-      </iframe>
+  </section>
+  <section class="margin-top-medium" aria-labelledby="section-reporting-bugs">
+    <div class="layout-grid">
+      <div class="col-4-large">
+        <h2>Something Fishy? Report it to the Team</h2>
```

Comment:
this is missing the id "section-reporting-bugs"

Reply:
ok I will update this and the PR and #455 to make this change